torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1009,177 +1030,190 @@ void InsertPrepackUnpack(script::Module& module) {
   }
 }
 
-void FoldPrepackedWeightIntoModule(
-    script::Module& module,
-    const std::string& method_name,
-    const script::Module& linear_params_module,
-    const script::Module& conv_params_module) {
-  auto method = module.get_method(method_name);
-  auto graph = method.graph();
-  GRAPH_DUMP(
-      "Before FoldPrepackWeightIntoModule: ",
-      graph);
+struct FoldPrepackedWeightIntoModuleHelper {
```

Comment:
there seems to be more changes which didn't match the description of the commit anymore, please split the changes next time